README.md

```diff
@@ -28,10 +28,35 @@ If you want to temporarily disable the skips while working on a test suite, you
 run the file with a shim that temporarily disables them:
 
 
-```ruby
-ruby -I../lib -rdisable_skip <filename_test.rb>
+```sh
+ruby -I../lib -rdisable_skip filename_test.rb
 ```
 
+In you would like to use the `make` tool to run a single test, while
+developing clock, for example, you can do something like this:
+
+```sh
+ASSIGNMENT=clock ARGS='-p' rerun make test-assignment
```

Comment:
I was thinking about removing it altogether, since you can use any other thing that you may use for continuous testing.